src/components/CalloutBanner.js

```diff
@@ -71,11 +73,12 @@ const CalloutBanner = ({
   children,
   className,
 }) => {
+  const intl = useIntl()
   return (
     <StyledCard className={className}>
       <Image
         fluid={image}
-        alt={`${title} image`}
+        alt={`${title} ${translateMessageId("image", intl)}`}
```

Comment:
I think this is a candidate for moving the alt text to the page component rather than in `CalloutBanner.js`

`{title} image` is likely to be a poor alt tag and may not translate well.

Reply:
Agreed. I think we should just keep this component "dumb" in the sense that it has no knowledge of `intl`. Let's keep that logic in the parent components & explicitly pass the `altText` as a prop.

In general, I think our approach should be to minimize the number of components that must instantiate the `intl` object with `useIntl()`